nilearn/plotting/tests/test_js_plotting_utils.py

```diff
@@ -295,11 +295,16 @@ def test_temp_file_removing():
     html = js_plotting_utils.HTMLDocument('hello')
     wb_open = webbrowser.open
     webbrowser.open = _open_mock
+    fd, tmpfile = tempfile.mkstemp()
     try:
+        os.close(fd)
+        assert_no_warnings(html.open_in_browser,
```

Comment:
The trouble with this approach is if in future some component called in this function's stack generates a warning, say a FutureWarning, then this test will fail for no reason.

Reply:
warnings are not meant to be systematically ignored. if this test fails because of a new warning we will fix the cause of the warning or change the test and add a comment explaining why the warning is acceptable.